.rubocop.yml

```diff
@@ -0,0 +1,38 @@
+require:
```

Comment:
I'm not against the reformatting but maybe it should be checked in the Travis config aswell? @neckhair what do you think?